android/modules/ui/src/java/ti/modules/titanium/ui/widget/tabgroup/TiUIAbstractTabGroup.java

```diff
@@ -386,6 +391,11 @@ public TabProxy getSelectedTab()
 		return ((TabGroupProxy) getProxy()).getTabList().get(this.tabGroupViewPager.getCurrentItem());
 	}
 
+	public void updateTitle(String title)
+	{
+		this.actionBar.setTitle(title);
```

Comment:
We need a `null` check in case we're using the "Theme.AppCompat.NoTitleBar" theme.